memory-db/Cargo.toml

```diff
@@ -1,27 +1,28 @@
 [package]
 name = "memory-db"
-version = "0.12.2"
+version = "0.12.3"
 authors = ["Parity Technologies <admin@parity.io>"]
 description = "In-memory implementation of hash-db, useful for tests"
 repository = "https://github.com/paritytech/trie"
 license = "Apache-2.0"
 
 [dependencies]
-heapsize = { version = "0.4", optional = true}
-hash-db = { path = "../hash-db", default-features = false, version = "0.12.2"}
+heapsize = { version = "0.4", optional = true }
+parity-util-mem = { version = "0.1" }
```

Comment:
`heapsize` was optional previously and now `parity-util-mem` is a required dependency. Does it break `non-std` builds?

Reply:
Yes, it would have (I added a commit). Thanks for seeing this :+1: